index.d.ts

```diff
@@ -0,0 +1,22 @@
+declare module 'knex-stringcase' {
+    import * as Knex from 'knex';
+
+    type StringCase = 
+        'camelcase' | 'capitalcase' | 'constcase' |
+        'cramcase' | 'decapitalcase' | 'dotcase' |
+        'enumcase' | 'lowercase' | 'pascalcase' |
+        'pathcase' | 'sentencecase' | 'snakecase' |
+        'spacecase' | 'spinalcase' | 'titlecase' |
+        'trimcase' | 'uppercase';
```

Comment:
I'm not sure all of these possible stringcases will remain consistent, if the stringcase library adds more of them then I'd need to update my library.